src/style.css

```diff
@@ -1 +1,74 @@
-/* add styles here */
+:root {
+  --custom-color: rgba(61, 78, 97, 1);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.search-form--large {
+  margin-bottom: 20px;
+  margin-top: 20px;
+}
+
+.search-form__input {
+  width: 100%;
+  border: 1px solid rgba(225, 231, 237, 1);
+  border-radius: 4px;
+  outline: none;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: var(--custom-color);
+}
+
+.search-form__input::placeholder {
+  color: var(--custom-color);
+}
+
+.search-form__input--large {
+  height: 70px;
+
+  padding-left: 62px;
+
+  background-size: 19px 19px;
+  background-position: 26px center;
+  font-size: 16px;
+}
+
+.search-form__input--small {
+  height: 42px;
+
+  padding-left: 33px;
+
+  background-size: 11px 11px;
+  background-position: 13px center;
+  font-size: 14px;
+}
+
+.search-form__input--large:hover,
+.search-form__input--small:hover {
+  font-family: Avenir-Book, sans-serif;
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.search-form__input--large:focus,
+.search-form__input--small:focus {
+  font-family: Avenir-Heavy, sans-serif;
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
```

Comment:
Your focused input's text seems to be missing a text shadow style as in the Figma Design; could you come up with a way to implement it

Yours:
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/52002856/96e07d2b-5012-4b2a-904d-2d2e8b4fa0ad)

Figma:
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/52002856/51e65e04-344b-41a5-9c63-9f28fd455d2d)

Reply:
Man, your eagle eyes are absolutely right. I confess I had noticed the shadow, but there were only specifications for box-shadow in Figma, not for text-shadow. I'll take the opportunity and create the shadow for the input of the small search bar.